viadot/tasks/genesys.py

```diff
@@ -10,8 +10,8 @@
 from prefect.engine import signals
 from prefect.utilities import logging
 from prefect.utilities.tasks import defaults_from_attrs
-from viadot.task_utils import *
 
+from viadot.task_utils import *
```

Comment:
Why this line is added here?

Reply:
Because if I try to call it directly a circle import failure raises. Actually, I didn't introduce it, it was made by Angelika when the new data frame check was added.